Solutions/Corelight/Hunting Queries/CorelightRarePOST.yaml

```diff
@@ -17,7 +17,7 @@ query: |
   corelight_http
   | where TimeGenerated > ago(24h)
   | where method in~ ('POST', 'PUT')
-  | where status_code >= 400 and status_code < 500
+  | where toint(status_code) >= 400 and toint(status_code) < 500

```

Comment:
why did you changed this? may be the data is coming for status code as int only?

Reply:
as per discussion we will keep this.